skycultures/arabian_peninsula/description.en.utf8

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+<title>Arabian Peninsula</title>
+</head>

```

Comment:
By formal rules no need header block for HTML'ed description, because we includes the descripting in our HTML page.